resources/services/fsx/filesystems.go

```diff
@@ -0,0 +1,563 @@
+						Type:        schema.TypeBigInt,
+						Resolver:    schema.PathResolver("DataRepositoryConfiguration.ImportedFileChunkSize"),
+					},
+					{
+						Name:        "data_repo_cfg_lifecycle",
+						Description: "Describes the state of the file system's S3 durable data repository, if it is configured with an S3 repository",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DataRepositoryConfiguration.Lifecycle"),
+					},
+					{
+						Name:        "deployment_type",
+						Description: "The deployment type of the FSx for Lustre file system",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "drive_cache_type",
+						Description: "The type of drive cache used by PERSISTENT_1 file systems that are provisioned with HDD storage devices",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "log_configuration_level",
+						Description: "The data repository events that are logged by Amazon FSx.  * WARN_ONLY - only warning events are logged.  * ERROR_ONLY - only error events are logged.  * WARN_ERROR - both warning events and error events are logged.  * DISABLED - logging of data repository events is turned off.  This member is required.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("LogConfiguration.Level"),
+					},
+					{
+						Name:        "log_configuration_destination",
+						Description: "The Amazon Resource Name (ARN) that specifies the destination of the logs",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("LogConfiguration.Destination"),
+					},
+					{
+						Name:        "mount_name",
+						Description: "You use the MountName value when mounting the file system",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "per_unit_storage_throughput",
+						Description: "Per unit storage throughput represents the megabytes per second of read or write throughput per 1 tebibyte of storage provisioned",
+						Type:        schema.TypeBigInt,
+					},
+					{
+						Name:        "root_squash_configuration_no_squash_nids",
+						Description: "When root squash is enabled, you can optionally specify an array of NIDs of clients for which root squash does not apply",
+						Type:        schema.TypeStringArray,
+						Resolver:    schema.PathResolver("RootSquashConfiguration.NoSquashNids"),
+					},
+					{
+						Name:        "root_squash_configuration_root_squash",
+						Description: "You enable root squash by setting a user ID (UID) and group ID (GID) for the file system in the format UID:GID (for example, 365534:65534)",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("RootSquashConfiguration.RootSquash"),
+					},
+					{
+						Name:        "weekly_maintenance_start_time",
+						Description: "The preferred start time to perform weekly maintenance, formatted d:HH:MM in the UTC time zone",
+						Type:        schema.TypeString,
```

Comment:
timestamp?

Reply:
I don't think so, as it is just a time of day, not a specific point in time.